### PR TITLE
chore: port 3709 to release 53.8.1

### PR DESCRIPTION
**Pull request by rcoringrato-sfdc**
### What does this PR do?

Port changes that remove log4j classes from Apex Language Server into patch release 53.8.1

### CODE REVIEW

File: packages/salesforcedx-vscode/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,19 @@
+# 53.8.0 - December 16, 2021
```
Comment (randi274) [inline packages/salesforcedx-vscode/CHANGELOG.md:1]: Does this need to be changed to 53.8.1?

File: SHA256.md [hunk 2]
```diff
@@ -8,24 +8,24 @@ make sure that their SHA values match the values in the list below.
    following the instructions at
    https://code.visualstudio.com/docs/editor/extension-gallery#_common-questions.
    For example, download,
-   https://salesforce.gallery.vsassets.io/_apis/public/gallery/publisher/salesforce/extension/salesforcedx-vscode-core/53.7.0/assetbyname/Microsoft.VisualStudio.Services.VSIXPackage.
+   https://salesforce.gallery.vsassets.io/_apis/public/gallery/publisher/salesforce/extension/salesforcedx-vscode-core/53.8.0/assetbyname/Microsoft.VisualStudio.Services.VSIXPackage.
```
Comment (randi274) [inline SHA256.md:11]: Should this be 53.8.1?

File: SHA256.md [hunk 3]
```diff
@@ -8,24 +8,24 @@ make sure that their SHA values match the values in the list below.
...
-d8d290bc203685f57ddb95fb1e2a57f87859ff4a518f03ae016f2577da184d33  salesforcedx-vscode-apex-replay-debugger-53.7.0.vsix
-42c2cc76e1c580b75089f9dbf5bead5e0787239bf3ad5f42576c34fb5459535e  salesforcedx-vscode-apex-53.7.0.vsix
-f188fbdccfa006c050ffa73a76c60315c16449c9b4b9ce928950e383417494e6  salesforcedx-vscode-core-53.7.0.vsix
-df8399134b6ea59bd9b9155f7c0f8ebcf9b882b936013157f47e05eb64075bff  salesforcedx-vscode-expanded-53.7.0.vsix
-2e3b4705ff888e15713246106ccf5921126cd0f4092df875615cc7571366298c  salesforcedx-vscode-lightning-53.7.0.vsix
-1535c51efdeb47058e7c3bb43297ac8b07815365754f83a69263600150dd1bc8  salesforcedx-vscode-lwc-53.7.0.vsix
-e56d7a763d502196767c02073319caa1af0e6751e6ca8115a44bea730a5bb383  salesforcedx-vscode-soql-53.7.0.vsix
-6e4f589be768d6763bf111f4ca7ca25585bac1afaad0d98d297359b045aad997  salesforcedx-vscode-visualforce-53.7.0.vsix
-9a252edb8f80a3fff2d6c841087201f0eb3bb938224ac8d8cb7404f37b835284  salesforcedx-vscode-53.7.0.vsix
+d5c703ab1b25fd58ac74874141e8ad689c3ee5913e4a198d705da387d0f2fc17  salesforcedx-vscode-apex-debugger-53.8.0.vsix
```
Comment (randi274) [inline SHA256.md:19]: Should these also be be 53.8.1?

**Review by randi274 — COMMENTED** [review]
Just a few questions on 53.8.1 vs 53.8.0 throughout

File: packages/salesforcedx-vscode/CHANGELOG.md (hunk 1 above)
Comment (rcoringrato-sfdc) [reply packages/salesforcedx-vscode/CHANGELOG.md:1]: @randi274 Thats the commit from last weeks release, I didn't update the changelog for the patch.

**Review by randi274 — APPROVED** [review]

**Pull request closed without merging**